scripts/4-test-libtensorflow-cc.sh

```diff
@@ -26,4 +26,8 @@ EXAMPLE_MOUNT="/example"
 CMD="./build-and-run.sh"
 
 echo "Testing libtensorflow_cc in ${IMAGE_CPP} ... "
-docker run --rm --gpus all -v ${EXAMPLE_DIR}:${EXAMPLE_MOUNT} -w ${EXAMPLE_MOUNT} ${IMAGE_CPP} ${CMD} | tee ${LOG_FILE}
+if [ "$ARCH" = "amd64" ]; then
+    docker run --rm --gpus all -v ${EXAMPLE_DIR}:${EXAMPLE_MOUNT} -w ${EXAMPLE_MOUNT} ${IMAGE_CPP} ${CMD} | tee ${LOG_FILE}
+elif [ "$ARCH" = "arm64" ]; then
```

Comment:
Can we additionally test for the availability of `--runtime nvidia`? Would then also add an else-clause that includes neither `--gpus` nor `--runtime`.

Reply:
Instead of duplicating the entire line, the if-else-clause could also only set a single variable that is then part of the not-duplicated `docker run` command below.

Reply:
I can't find a good solution testing for the availability of `--runtime nvidia` or `--gpus all`.

But in commit [524ff24](https://github.com/ika-rwth-aachen/libtensorflow_cc/pull/2/commits/524ff24359332ab6b45889de9e69975e59c66675) I have made adjustments to avoid code duplication.